.github/workflows/push.yml

```diff
@@ -67,7 +67,16 @@ jobs:
           make vendor
           pip3 install wheel==0.45.1
 
+      - name: Run tests without coverage
```

Comment:
Nit: "without coverage" is implied and clarified by the presence of a "with coverage" step.